Adding skimage regionprops support to get_radial_distribution

Skimage's regionprops uses boolean label images internally to send to extra methods.
To add compatibility to regionprops, a small conversion from boolean to int within get_radial_distribution is necessary because at least centrosome.cpmorphology.distance_to_edge() cannot handle boolean labels.

```
src/squidpy/im/_feature.py:250: in quantify_morphology
    region_props = _get_region_props(
src/squidpy/im/_feature.py:97: in _get_region_props
    extracted_props[prop].append(getattr(region, prop))
../../miniconda3/envs/mamba/envs/squidpy-dev/lib/python3.10/site-packages/skimage/measure/_regionprops.py:353: in __getattr__
    multichannel_list = [func(self.image,
../../miniconda3/envs/mamba/envs/squidpy-dev/lib/python3.10/site-packages/skimage/measure/_regionprops.py:353: in <listcomp>
    multichannel_list = [func(self.image,
src/squidpy/im/_measurements.py:483: in radial_distribution
    d_to_edge = centrosome.cpmorphology.distance_to_edge(labels)
../../miniconda3/envs/mamba/envs/squidpy-dev/lib/python3.10/site-packages/centrosome/cpmorphology.py:4385: in distance_to_edge
    colors = color_labels(labels)
../../miniconda3/envs/mamba/envs/squidpy-dev/lib/python3.10/site-packages/centrosome/cpmorphology.py:4084: in color_labels
    v_count, v_index, v_neighbor = find_neighbors(dt_labels)
../../miniconda3/envs/mamba/envs/squidpy-dev/lib/python3.10/site-packages/centrosome/cpmorphology.py:3973: in find_neighbors
    adjacent_mask = adjacent(labels)
../../miniconda3/envs/mamba/envs/squidpy-dev/lib/python3.10/site-packages/centrosome/cpmorphology.py:169: in adjacent
    if high > np.iinfo(labels.dtype).max:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = iinfo(min=-128, max=127, dtype=bool), int_type = dtype('bool')

    def __init__(self, int_type):
        try:
            self.dtype = numeric.dtype(int_type)
        except TypeError:
            self.dtype = numeric.dtype(type(int_type))
        self.kind = self.dtype.kind
        self.bits = self.dtype.itemsize * 8
        self.key = "%s%d" % (self.kind, self.bits)
        if self.kind not in 'iu':
>           raise ValueError("Invalid integer data type %r." % (self.kind,))
E           ValueError: Invalid integer data type 'b'.
```

--- a/src/cp_measure/fast/measureobjectintensitydistribution.py
+++ b/src/cp_measure/fast/measureobjectintensitydistribution.py
@@ -132,6 +132,9 @@ def get_radial_distribution(
         radius will be counted in an overflow bin. The radius is measured in pixels.
     """
 
+    if labels.dtype == bool:
+        labels = labels.astype(numpy.integer)
+
     unique_labels = numpy.unique(labels)
     nobjects = len(unique_labels[unique_labels>0])
     d_to_edge = centrosome.cpmorphology.distance_to_edge(labels)

